### PR TITLE
dev-only HTML source formatting

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/4906

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
 beautifulsoup4
 boto3
-Django==2.2.14
+Django==2.2.13
```
Comment (Pomax) [inline requirements.in:3]:
```suggestion
Django==2.2.14
```

File: network-api/networkapi/utility/middleware.py [hunk 2]
```diff
@@ -12,6 +13,22 @@ class HttpResponseTemporaryRedirect(HttpResponseRedirectBase):
...
 
+class HTMLCleanUpMiddleware:
+    """
+    See http://blog.christianperone.com/2009/10/beautiful-django/
+    """
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.get_response(request)
```
Comment (cadecairos) [inline network-api/networkapi/utility/middleware.py:24]: decoding the response is side-stepping the gzip compression middleware. When the beautified HTML is set as the new value for `response.content` below, it's no longer gzip compressed, and clients choke trying to decode it. (Tested in Firefox and using httpie)

Comment (cadecairos) [reply network-api/networkapi/utility/middleware.py:24]: We can either find a way to re-compress the payload (and recalculate the `Content-Length` value) or we can disable GZip compression when DEBUG is enabled.

**Comment by Pomax** [comment]
@cadecairos gzip _has_ to be the loaded first in the set of middleware that mess with responses (https://docs.djangoproject.com/en/3.0/ref/middleware/#module-django.middleware.gzip, "This middleware should be placed before any other middleware that need to read or write the response body so that compression happens afterward.") so I moved it to the back of the line, and it looks like everything's running fine now?

**Pull request closed without merging**